build.override.targets

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="WcfSetup" Condition="'$(WithCategories)' == 'OuterLoop'" BeforeTargets="Test">
+    <Exec Command="$(MySetupScript) > $(MyLogFile)" />
+    <ReadLinesFromFile File="$(MyLogFile)" >
```

Comment:
Because if the ExitCode is anything other than 0 it is a build error. I wasn't able to take a non-0 code and set it to a property without it still throwing a build error.